### PR TITLE
readme: update LTS plan detail

### PR DESCRIPTION
**Pull request by jasnell**
Update LTS plan detail. Remove the "Proposed" tag, add detail about 
named LTS releases.
/cc @nodejs/lts .. quick review please before I land?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,24 @@ The Proposed LTS Plan is:
...
     lead to *semver-major* changes landing within an LTS stream, such
     situations will be rare and will land as *semver-minor* bumps.
+11. All LTS releases will be assigned a "codename" drawn from the names of
+    elements on the Periodic Table of Elements. For each upcoming LTS
+    release, the LTS Working Group will select a handful of candidate names
+    and submit those for a collaborator vote.
+
+## Node abstraction layer
+
+It should be stated that the abstraction layer (currently `nan`) should
```
Comment (Fishrock123) [inline README.md]: Isn't it capital `NAN`?

**Comment by Fishrock123** [comment]
Ya lgtm

File: README.md (hunk 1 above)
Comment (rvagg) [inline README.md]: yes, NAN

**Comment by rvagg** [comment]
lgtm sans comment

**Comment by jasnell** [comment]
Ok, fixed. Merging!

File: README.md [hunk 2]
```diff
@@ -41,9 +41,24 @@ The Proposed LTS Plan is:
...
+It should be stated that the abstraction layer (currently `NAN`) should
+support all *current* LTS releases. Given that Active LTS will overlap
+for a period of 6 months, this means that the abstraction layer will, at
+any given point in time, fully support a maximum of 2 LTS releases.
 
 ## Example
 
+The specific details may vary depending on the `next` and `master` release plan
+that is ultimately adopted. However, the release plan will not impact the
+schedule or proceses for managing an LTS release once it has been cut.
```
Comment (thefourtheye) [inline README.md]: "processes"

**Pull request merged**